parser/shared/src/main/scala/hmda/parser/fi/lar/LarParsingError.scala

```diff
@@ -0,0 +1,4 @@
+package hmda.parser.fi.lar
+
+case class LarParsingError(lineNumber: Int = 0, errorMessages: List[String] = List.empty)
+case class ParsingErrorSummary(transmittalSheetErrors: Seq[List[String]] = Seq.empty, larErrors: Seq[LarParsingError] = Seq.empty)
```

Comment:
@kgudel is right. Currently the output doesn't quite match the documentation because there's an extra layer of `[]`s inside of `"transmittalSheetErrors"`, like so:
```json
  "transmittalSheetErrors": [
    [
      "Incorrect number of fields. found: 39, expected: 21"
    ]
  ],
```